assets/js/script.js

```diff
@@ -158,18 +165,19 @@ $(document).on('click', '.js-letter', function(evt) {
   }
 })
 
-$(document).on('click', '.js-tab', function(evt) {
-  if ($(this).data('index') > -1) {
+$(document).on('click', '.js-tab-back', function(evt) {
+  let tab = $(this).children('.js-tab')
```

Comment:
better to use `var` -- `let` isn't supported across all browsers yet without babel.